src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -91,22 +91,26 @@ bool isCreateConstraintActive(Gui::Document *doc)
 // Utility method to avoid repeating the same code over and over again
 void finishDistanceConstraint(Gui::Command* cmd, Sketcher::SketchObject* sketch, bool isDriven=true)
 {
+
+    ParameterGrp::handle hGrp = App::GetApplication().GetParameterGroupByPath("User parameter:BaseApp/Preferences/Mod/Sketcher");
+
     // Get the latest constraint
     const std::vector<Sketcher::Constraint *> &ConStr = sketch->Constraints.getValues();
     Sketcher::Constraint *constr = ConStr[ConStr.size() -1];
 
     // Guess some reasonable distance for placing the datum text
     Gui::Document *doc = cmd->getActiveGuiDocument();
     float sf = 1.f;
+    float lp = hGrp->GetFloat("RaDiaAngle", 15) * (M_PI / 180);
```

Comment:
I'd be in favor of a longer parameter name here instead of "RaDiaAngle" -- it's being assigned to a pretty opaque variable, I feel like a future reader is going to have no idea what this means. I'd go so far as "RadiusDiameterConstraintAngle" or something similar.

Reply:
@chennes Thx for review. I have to agree that this one wasn't very good. I updated with a proposal that tries to keep it reasonably short while being more explaining. I also added a comment.
Also keep in mind that these switches are documented in https://wiki.freecadweb.org/Fine-tuning when merged ;)